components/business-adaptors/sap/src/main/java/org/wso2/carbon/transports/sap/SAPTransportSender.java

```diff
@@ -229,9 +236,10 @@ private String evaluateRFCfunction(JCoFunction function, JCoDestination destinat
 
         }
         // there seems to be some error that we need to report: TODO ?
-        if (returnStructure != null && (!(returnStructure.getString("TYPE").equals("")
-                                          || returnStructure.getString("TYPE").equals("S")))) {
-            throw new AxisFault(returnStructure.getString("MESSAGE"));
+        if (escapeErrorHandling == null || "".equals(escapeErrorHandling) || "false".equals(escapeErrorHandling)) {
```

Comment:
Shall we explain, why we need this conditions and the purpose of this in a comment?

Reply:
Comment added